.github/workflows/build.yml

```diff
@@ -0,0 +1,39 @@
+---
+    name: Build
+    on:
+      push:
+        branches:
+          - master
+
+    jobs:
+      build:
+        name: Build
+        runs-on: ubuntu-latest
+        steps:
+          - uses: actions/checkout@v2
+          - env:
+              TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            run: |
+              sudo apt update
+              sudo apt install -y gcc
+              sudo apt install -y libsodium-dev
+
+              curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs -o rustup.sh
+              sh ./rustup.sh -y
+              source $HOME/.cargo/env
+
+              cargo install cross
+
+              echo "${TOKEN}" | docker login https://docker.pkg.github.com \
+                -u "$GITHUB_REPOSITORY_OWNER" --password-stdin
+              
+              sudo apt install -y gnupg2
+
+              gpg2 --keyserver hkp://pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
```

Comment:
what is this needed for?

Reply:
I think it's required to install rvm
https://rvm.io/rvm/install

ruby can also be installed without rvm if necessary